src/modules/common/containers/drag-and-drop-target.jsx

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class DragAndDropTarget extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      canDrop: false,
+    };
+
+    this.handleDragEnter = this.handleDragEnter.bind(this);
+    this.handleDragOver = this.handleDragOver.bind(this);
+    this.handleDragLeave = this.handleDragLeave.bind(this);
+    this.handleDrop = this.handleDrop.bind(this);
+  }
+
+  handleDragEnter(e) {
+    e.preventDefault();
+    this.setState({ canDrop: true });
+    this.props.onDragEnter(...arguments);
```

Comment:
I think the key differences between this and the [Zooniverse React Components version](https://github.com/zooniverse/Zooniverse-React-Components/blob/master/src/components/drag-and-drop-target.jsx) is how `arguments` are handled. I attempted all sorts of variations to avoid doing as is, but none worked, though I feel like might be making simple syntax mistake.

Reply:
I think I may have been following the eslint suggestion: https://eslint.org/docs/rules/prefer-rest-params
But it doesn't appear to work? I don't recall testing the ZRC version manually and the unit tests are missing tests checking the test spy arguments: https://github.com/zooniverse/Zooniverse-React-Components/blob/master/test/drag-and-drop-target.test.js

I'd prefer the ZRC version get fixed than to have a copy of almost the same thing here.